doc/dev/coding-guidelines-and-review.rst

```diff
@@ -109,6 +109,7 @@ ready for review::
      http://en.wikipedia.org/wiki/Changelog#Format
    - [ ] Was a spellchecker run on the source code and documentation after
      changes were made?
+   - [ ] Does the script respect streaming IO? (Is it tested for streaming IO?)
```

Comment:
Some pull requests don't have scripts, therefore 'the script' doesn't make sense.